src/lib/components/workspace/member-info/MemberInfo.svelte

```diff
@@ -0,0 +1,188 @@
+<script lang="ts">
+  import MemberDropdown from "$lib/components/dropdown/MemberDropdown.svelte";
+  import MemberWorkspace from "../member-worspace/MemberWorkspace.svelte";
+  import { TeamRole, WorkspaceRole } from "$lib/utils/enums/team.enum";
+  import type {
+    TeamServiceMethods,
+    userDetails,
+    workspaceDocumentWithPosition,
+  } from "$lib/utils/interfaces";
+  import type { MemberPopType } from "$lib/utils/types/common.type";
+
+  export let user: userDetails;
+  export let teamRole: TeamRole;
+  export let workspaces: workspaceDocumentWithPosition[];
+  export let hasPermission: boolean = false;
```

Comment:
no need to explicitely use : boolean when you are assigning a primitive data type, false

Reply:
resolved